### PR TITLE
Updates in line with Technical Annex (CCS parameters)

### PR DESCRIPTION
**Pull request by Certiman**
Please note I also used:
- `dcterms:source` for the "Reference", i.e the EULEX or ERA document describing the basis of the parameter
- `rdfs:requires` for the "Applicable" - conditions
- `dcterms:relation` to link to another parameter if there is a dependency
- `dcterms:description` for the "Explanations" or any data not in Defintion/Data presentation.

The `dcterms`-prefix was therefore also added.

### CODE REVIEW

File: ontology.ttl [hunk 1]
```diff
@@ -5455,30 +5674,35 @@ era:maximumDesignSpeed rdf:type owl:DatatypeProperty ;
...
                                              "1.2.1.1.3.2.1" ;
                                era:unitOfMeasure <https://qudt.org/vocab/unit/A-PER-M> ;
                                <http://purl.org/dc/terms/created> "2023-03-14"^^xsd:date ;
                                <http://purl.org/dc/terms/modified> "2024-04-18"^^xsd:date ;
-                               rdfs:comment """Maximum interference current limits allowed for track circuits for a defined frequency band (to be expressed in A/m).
-                               TODO: update the range to xsd:double with a unit description"""@en ;
+                               rdfs:comment """Maximum interference current limits allowed for track circuits for a defined frequency band (to be expressed in A/m)."""@en ;
                                rdfs:isDefinedBy <http://data.europa.eu/949/> ;
-                               rdfs:label "Maximum interference current"@en ;
+                               dcterms:source "https://www.era.europa.eu/system/files/2023-09/index077_-_ERA_ERTMS_033281_v5.pdf" ;
```
Comment (gatemezing) [inline ontology.ttl:5687]: dcterms:source should point to an URI, so change it to be` dcterms:source <https://www.era.europa.eu/system/files/2023-09/index077_-_ERA_ERTMS_033281_v5.pdf>`

File: ontology.ttl [hunk 2]
```diff
@@ -5498,11 +5722,14 @@ era:maximumLocomotivesCoupled rdf:type owl:DatatypeProperty ;
 era:maximumMagneticFieldDirectionX rdf:type owl:DatatypeProperty ;
                                    rdfs:domain era:MaximumMagneticField ;
                                    rdfs:range xsd:integer ;
+                                   era:XMLName "CCD_ACMagFieldMax" ;
                                    era:rinfIndex "1.1.1.3.4.2.3" ,
                                                  "1.2.1.1.3.2.3" ;
                                    <http://purl.org/dc/terms/created> "2023-03-14"^^xsd:date ;
                                    rdfs:comment "The maximum magnetic field limits allowed for axle counters (in dB µA/m) for a defined frequency band. Direction X."@en ;
                                    rdfs:isDefinedBy <http://data.europa.eu/949/> ;
+                                   dcterms:source "https://www.era.europa.eu/system/files/2023-09/index077_-_ERA_ERTMS_033281_v5.pdf" ;
```
Comment (gatemezing) [inline ontology.ttl:5731]: same as above for dcterms:source

File: ontology.ttl [hunk 3]
```diff
@@ -5524,11 +5754,14 @@ era:maximumMagneticFieldDirectionY rdf:type owl:DatatypeProperty ;
 era:maximumMagneticFieldDirectionZ rdf:type owl:DatatypeProperty ;
                                    rdfs:domain era:MaximumMagneticField ;
                                    rdfs:range xsd:integer ;
+                                   era:XMLName "CCD_ACMagFieldMax" ;
                                    era:rinfIndex "1.1.1.3.4.2.3" ,
                                                  "1.2.1.1.3.2.3" ;
                                    <http://purl.org/dc/terms/created> "2023-03-14"^^xsd:date ;
                                    rdfs:comment "The maximum magnetic field limits allowed for axle counters (in dB µA/m) for a defined frequency band. Direction Z."@en ;
                                    rdfs:isDefinedBy <http://data.europa.eu/949/> ;
+                                   dcterms:source "https://www.era.europa.eu/system/files/2023-09/index077_-_ERA_ERTMS_033281_v5.pdf" ;
```
Comment (gatemezing) [inline ontology.ttl:5763]: range of dcterms:source a URI

File: ontology.ttl [hunk 4]
```diff
@@ -498,20 +500,21 @@ era:cantDeficiencyBasicSSP rdf:type owl:ObjectProperty ;
...
                            rdfs:comment """Essential information for drivers of trains with a worse (lower) tolerated cant deficiency than those for which the ETCS trackside provides SSP (Static Speed Profiles) in conjunction with  parameter \"Other Cant Deficiency train categories for which the ETCS trackside is configured to provide SSP\".
 
-According to the specification referenced in Appendix A-1, index [C]:
-
 Subset-026 (3.11.3.2.1.1) definition:
 a) The “Cant Deficiency” SSP categories: the cant deficiency value assigned to one category shall define the maximum speed, determined by suspension design, at which a particular train can traverse a curve and thus can be used to set a specific speed limit in a curve with regards to this category.
 """@en ;
                            rdfs:isDefinedBy <http://data.europa.eu/949/> ;
+                           dcterms:requires "Only applicable when selected value for 1.1.1.3.2.1 (ETCS present)."@en ;
+                           dcterms:source "According to the specification referenced in Appendix A-1, index [C]"@en ;
```
Comment (gatemezing) [inline ontology.ttl:517]: dcterms:source should be a URI

File: ontology.ttl [hunk 5]
```diff
@@ -889,6 +897,7 @@ era:etcsBaseline rdf:type owl:ObjectProperty ;
                  <http://purl.org/dc/terms/modified> "2021-08-31"^^xsd:date ,
                                                      "2024-04-18"^^xsd:date ;
                  rdfs:comment "ETCS baseline installed lineside."@en ;
+                 dcterms:source "TBD"@en ;
```
Comment (gatemezing) [inline ontology.ttl:900]: Let's avoid "TBD3 if possible

File: ontology.ttl [hunk 6]
```diff
@@ -945,6 +954,7 @@ era:etcsInfill rdf:type owl:ObjectProperty ;
                                                    "2024-04-18"^^xsd:date ;
                rdfs:comment "Information about installed trackside equipment capable to transmit infill information by loop or Global System for Mobile communications for Railways (GSM-R) for level 1 installations."@en ;
                rdfs:isDefinedBy <http://data.europa.eu/949/> ;
+               dcterms:source "TSI CCS 4.2.2"@en ;
```
Comment (gatemezing) [inline ontology.ttl:957]: dcterms;source should be a URI

File: ontology.ttl [hunk 7]
```diff
@@ -989,10 +1010,12 @@ era:etcsMVersion rdf:type owl:ObjectProperty ;
                                "1.2.1.1.1.10" ;
                  era:usedInRCCCalculations "true"^^xsd:boolean ;
                  <http://purl.org/dc/terms/created> "2021-08-08"^^xsd:date ;
-                 <http://purl.org/dc/terms/modified> "2021-09-12"^^xsd:date ;
+                 <http://purl.org/dc/terms/modified> "2021-09-12"^^xsd:date , "2024-04-18"^^xsd:date ;
                  rdfs:comment "ETCS M_version according to the specification referenced in Appendix A-1, index [C]."@en ;
                  rdfs:isDefinedBy <http://data.europa.eu/949/> ;
                  rdfs:label "ETCS M_version"@en ;
+                 dcterms:requires "Only applicable when selected value for 1.1.1.3.2.1 (ETCS present)."@en ;
+                 dcterms:source "According to the specification referenced in Appendix A-1, index [C]"@en ;
```
Comment (gatemezing) [inline ontology.ttl:1018]: same comment for dcterms;source range as URI

File: ontology.ttl [hunk 8]
```diff
@@ -1013,6 +1036,8 @@ era:etcsSystemCompatibility rdf:type owl:ObjectProperty ;
                             <http://purl.org/dc/terms/created> "2020-08-31"^^xsd:date ;
                             <http://purl.org/dc/terms/modified> "2021-09-12"^^xsd:date ;
                             rdfs:comment "ETCS requirements used for demonstrating technical compatibility."@en ;
+                            dcterms:source "TBD"@en ;
+                            dcterms:definition "TBD"@en ;
```
Comment (gatemezing) [inline ontology.ttl:1040]: let's avoid TBD

File: ontology.ttl [hunk 9]
```diff
@@ -4897,8 +5109,14 @@ era:imCode rdf:type owl:DatatypeProperty ;
...
-           rdfs:comment "(deprecated) Infrastructure manager means anybody or undertaking that is responsible in particular for establishing and maintaining railway infrastructure or a part thereof."@en ;
+	         dcterms:description """The Code is a unique identifier for the Infrastructure Manager and it shall be verified on national level. 
+- If the IM is subject to TAF/TAP TSIs, it corresponds to the code used in TAF/TAP TSIs. 
+- In other cases, it corresponds to the "organisation code” assigned by the Agency for the specific needs of the RINF
+Each Section of Line may concern only one IM. """@en ; 
+           rdfs:comment """(deprecated) Infrastructure manager means any body or firm responsible in particular for establishing, managing and maintaining railway infrastructure, including traffic management and control-command and signalling; 
+the functions of the infrastructure manager on a network or part of a network may be allocated to different bodies or firms."""@en ;
            rdfs:isDefinedBy <http://data.europa.eu/949/> ;
+	         rdfs:seeAlso "http://data.europa.eu/eli/dir/2012/34/oj (Article 3(2))." ;
```
Comment (gatemezing) [inline ontology.ttl:5119]: s/rdfs:seeAlso "http://data.europa.eu/eli/dir/2012/34/oj (Article 3(2))."/` rdfs:seeAlso <http://data.europa.eu/eli/dir/2012/34/oj (Article 3(2))>`

**Review by gatemezing — CHANGES_REQUESTED** [review]
See the comments in the reviw

**Comment by Certiman** [comment]
@gatemezing , thanks for a second review.

**Review by gatemezing — APPROVED** [review]
Good to go :)

**Pull request merged**